### DISCUSSION TITLE
Not able to load slide

### DISCUSSION THREAD

**Discussion by SanjanaSogimatt** [camicroscope/caMicroscope#897] [Q&A]
I tired to open the .svs file which I uploaded but I am getting an error message

![file-not-updating](https://github.com/camicroscope/caMicroscope/assets/87171143/d361167e-41d8-43c3-be77-5cc6111341ca)
@birm can you help me with this?

**Comment by harshpreet14** [comment]
Hey Sanjana ! Are you running it using Docker or Live Server ?

**Comment by birm** [comment]
Also can you paste the full url of what its trying to open? Also check `docker ps -a` for ca-iip

**Comment by SanjanaSogimatt** [comment]
Well, it's strange but I rebuilt the project and it was fixed on it's own
